packages/@react-aria/overlays/src/Overlay.tsx

```diff
@@ -56,7 +57,7 @@ export function Overlay(props: OverlayProps): ReactNode | null {
   let contextValue = useMemo(() => ({contain, setContain}), [contain, setContain]);
 
   let {getContainer} = useUNSTABLE_PortalContext();
-  if  (!props.portalContainer && getContainer) {
+  if (getContainer) {
```

Comment:
I'm a teeny bit apprehensive about preferring the context over the prop. This would actually change how an app works if they were using both.
Probably better to let the local prop override the context.